hs_csdms/migrations/0002_auto_20200626_2152.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-06-25 17:46
+from __future__ import unicode_literals
+
+from django.db import migrations
+from hs_csdms.utils import split_csdms_names
+
+
+def populate_csdms_names(apps, schema_editor):
+    """ Store each part from split_csdms into LDAWord model
+    """
+    csdms_names, csdms_decors, csdms_measures = split_csdms_names()
+    CSDMSName = apps.get_model('hs_csdms', 'CSDMSName')
+    for csdms_name in csdms_names:
+        if len(csdms_name) > 1:
+            csdms_record = CSDMSName()
+            csdms_record.source = 'CSDMSName'
```

Comment:
I assume there are plans to add other sources eventually?

Reply:
Yes. We might add words from other sources in the future.